### PR TITLE
feat(alpha): support RDF response via http query request (#8004) 

### LINKED ISSUE #8529: [ENHANCEMENT]: Add support for rdf response via http
### What you wanted to do.

v21.12 PR https://github.com/dgraph-io/dgraph/pull/8004

### PR DESCRIPTION
**Pull request by all-seeing-code**
Fixes #8529
Closes: DGRAPHCORE-223

inspired from https://github.com/dgraph-io/dgraph/pull/8004

### CODE REVIEW

**Review by mangalaman93 — DISMISSED** [review]

**Review by sanjayk-github-dev — DISMISSED** [review]
The changes LGTM

File: dgraph/cmd/alpha/http_test.go [hunk 1]
```diff
@@ -985,3 +986,54 @@ func TestQueryBackwardCompatibleWithGraphqlPlusMinusHeader(t *testing.T) {
...
+	  }
+	}`
+	resp, _, err := queryWithTs(queryInp{body: q1, typ: "application/dql", respFmt: "rdf"})
+	require.NoError(t, err)
+	require.Equal(t, `{"data":""}`, resp)
+
+	m1 := `
+    {
+	  set {
+		_:alice <name> "Bob \"<the builder>\"" .
```
Comment (MichelDiz) [inline dgraph/cmd/alpha/http_test.go:1009]: Minor: This blank node is confusing. It says "Alice" but in the value there is Bob...

**Review by MichelDiz — DISMISSED** [review]
The response looks good

```
{
    "data":"<0x1> <name> \"Bob \\\"\\u003cthe builder\\u003e\\\"\" .\n<0x1> <balance> \"110\" .\n"
}

File: dgraph/cmd/alpha/http_test.go (hunk 1 above)
Comment (mangalaman93) [reply dgraph/cmd/alpha/http_test.go]: Thanks for noticing Michel. Fixed it now.

**Review by MichelDiz — APPROVED** [review]

**Review by sanjayk-github-dev — APPROVED** [review]

**Pull request merged**